canonical-web-and-design/jaas-dashboard
# Update dockerfile to use serve instead of http.server

## Done

This change allows reviewers to refresh the application at any route and have the correct assets served for that path instead of having it 404 on all paths but `/`.

## QA

- Visit the demo link and navigate around the application.
- Refresh at various points and it should load the app to those pages.
